### PR TITLE
Use Unicode escape sequences for control characters

### PR DESCRIPTION
**Pull request by cston**
Allow control characters to be displayed as Unicode escape sequences, particularly in quoted characters and strings in the EE.

### CODE REVIEW

**Comment by cston** [comment]
@KevinH-MS, @amcasey, @tmat, @ManishJayaswal

**Comment by gafter** [comment]
Is there a description of the purpose of this change?

**Comment by jaredpar** [comment]
@dotnet-bot test this please

**Comment by cston** [comment]
Description: Allow control characters to be displayed as Unicode escape sequences, particularly in quoted characters and strings in the EE.

File: src/Compilers/CSharp/Portable/SymbolDisplay/ObjectDisplay.cs [hunk 1]
```diff
@@ -225,14 +148,95 @@ internal static string FormatLiteral(bool value)
...
+            else
             {
-                var builder = pooledBuilder.Builder;
                 builder.Append(c);
             }
         }
 
         /// <summary>
+        /// Returns true if the character should be replaced and sets
+        /// replaceWith to the replacement text if the character is
```
Comment (VladimirReshetnikov) [inline src/Compilers/CSharp/Portable/SymbolDisplay/ObjectDisplay.cs]: paramref

Comment (cston) [inline src/Compilers/CSharp/Portable/SymbolDisplay/ObjectDisplay.cs]: Done.

**Pull request merged**